hooks/pre/31-mongo_storage_engine.rb

```diff
@@ -0,0 +1,51 @@
+if answers['katello']
+  MONGO_DB_NAME = param('katello', 'pulp_db_name')
+else
+  MONGO_DB_NAME = param('foreman_proxy_content', 'pulp_db_name')
+end
+
+def migration
+  export_dir = '/var/tmp/engine_upgrade'
```

Comment:
`/var/tmp` and `/var/lib` can different filesystems. Do we want to check for that?